.changeset/tiny-emus-rush.md

```diff
@@ -0,0 +1,17 @@
+---
+'@keystonejs/benchmarks': patch
+'@keystonejs/fields': patch
+'@keystonejs/test-utils': patch
+---
+
+**Removed usage of `graphqlRequest` and replaced it with corresponding utils functions from `server-side-graphql-client`**.
```

Comment:
I think we can be more concise with the changeset here. Keep in mind that the reason we write the changeset is so that we can have a changelog file generated. The reason we have a changelog is so that users of the package can understand what impact an upgrade might have on their project. For these packages, all the user really needs to know is that we 

```
Refactored internals to use `server-side-graphql-client`.
```

The generated changelog will contain a link to this PR, so if they want to know more details about what changed, they can follow that link and look at this diff.